packages/sindarian-i18n-cli/tsup.config.ts

```
@@ -0,0 +1,23 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig([
+  {
+    entry: { index: 'src/index.ts' },
+    format: ['esm'],
+    target: 'node18',
+    dts: true,
+    clean: true,
+    splitting: false,
```

ESLint

[error] Delete `,`

(prettier/prettier)